epytope/tutorials/data/allele_probabilities_europe.csv

```diff
@@ -65,7 +65,7 @@ HLA-B*73:01	0.000548847
 HLA-B*35:04	0.001646542
 HLA-A*02:17	0.001094092
 HLA-B*07:20	0.000274424
-HLA-A*24:09N	0.000273523
+HLA-A*24:09	0.000273523
```

Comment:
Threw error otherwise. I guess it is a typo

Reply:
It should be valid since it is a so called "'Null' allele - an allele that is not expressed" (http://hla.alleles.org/nomenclature/naming.html). Do you remember where it failed or what the error was exactly?

Reply:
I think I went through the tutorials and once there was an error and after I corrected that N it worked fine. I can check this if the tutorials are on the menu. We can leave it in the previous version if you want to :)